dashboard/legacy/middleware/files_api.rb

```diff
@@ -268,7 +268,6 @@ def get_file(endpoint, encrypted_channel_id, filename, code_projects_domain_root
     metadata = result[:metadata]
     abuse_score = [metadata['abuse_score'].to_i, metadata['abuse-score'].to_i].max
     not_found if abuse_score >= SharedConstants::ABUSE_CONSTANTS.ABUSE_THRESHOLD && !can_view_abusive_assets?(encrypted_channel_id)
-    not_found if profanity_privacy_violation?(filename, result[:body]) && !can_view_profane_or_pii_assets?(encrypted_channel_id)
```

Comment:
The only project type that is currently checked for profanity/privacy is Play Lab, so removing this exception.